src/google-maps/google-map/google-map.ts

```diff
@@ -227,7 +228,12 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
      * @deprecated `platformId` parameter to become required.
      * @breaking-change 10.0.0
      */
-    @Optional() @Inject(PLATFORM_ID) platformId?: Object) {
+    @Optional() @Inject(PLATFORM_ID) platformId?: Object,
+    /**
+     * @deprecated `_ngZone` parameter to become required.
+     * @breaking-change 11.0.0
+     */
+    private _ngZone?: NgZone) {
```

Comment:
Since this is a fix, we can make this required and cherrypick it into `9.0.x`. Changes for newly added code can still modify the API during RC if it's to fix a bug.